src/Lykke.Service.BitcoinCash.API.AzureRepositories/Wallet/ObservableWalletRepositorycs.cs

```diff
@@ -55,6 +55,11 @@ public async Task<(IEnumerable<IObservableWallet>, string ContinuationToken)> Ge
             return await _storage.GetDataWithContinuationTokenAsync(take, continuationToken);
         }
 
+        public async Task<IEnumerable<IObservableWallet>> GetAll()
```

Comment:
Better to make it with continuation token and do not read all wallets into the memory at once in the wallets importing tool